TextInputKit/TextInputKit/Code/AppKitExtensions/NSText+TextInputKit.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NSText+TextInputKit.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 19/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import AppKit
+
+extension NSText {
+
+    var textInputKit_selectedUtf16IntRange: Range<Int>? {
```

Comment:
Well, the only question is what prefixes to use: shortened (like "tik") or full (like "textInputKit").
I would rather shortened, but it's totally up to you.